PhenomRaidTools/Source/Utils.lua

```diff
@@ -18,7 +18,7 @@ local classColors = {
 }
 
 -- Create local copies of API functions which we use
-local UnitClass, GetSpellInfo, UnitExists, UnitIsDead, UnitName = UnitClass, GetSpellInfo, UnitExists, UnitIsDead, UnitName
+local UnitClass, GetSpellInfo, UnitExists, UnitIsDead, UnitName, UnitIsConnected, UnitIsDeadOrGhost = UnitClass, GetSpellInfo, UnitExists, UnitIsDead, UnitName, UnitIsConnected, UnitIsDeadOrGhost
```

Comment:
Would be good to add those to .luacheckrc aswell so it won't be marked as warning all the time.

Reply:
fixed in latest commit